src/components/Modal/ModalContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { v4 } from 'uuid';
+
+import { useInput } from '../../hooks/useInput';
+import { ModalContainerProps } from './interfaces';
+import { Modal } from './Modal';
+
+export const ModalContainer: React.FC<ModalContainerProps> = ({
+  onClose,
+  addTodo,
+}) => {
+  const [title, handleTitle] = useInput();
+  const [description, handleDescription] = useInput();
+  const createTodo = (title: string, description: string) => {
```

Comment:
Add an empty line before function. Don't save the place on the screen

Reply:
done